tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/FakeCommandTests.cs

```diff
@@ -23,50 +21,26 @@ public FakeCommandTests(ITestOutputHelper output)
         [Trait("Category", "EndToEnd")]
         public void SubmitsTracesWithNetStandard()
```

Comment:
While you're here:

```suggestion
        public void SubmitsTraces()
```

Reply:
Thanks! Fixed in a7e17b37861b66da579c5108f074abe9e294420e.